src/grid/index.tsx

```diff
@@ -1,31 +1,31 @@
+import './utils';
+
 import WidgetBase from '@dojo/framework/core/WidgetBase';
-import { v, w } from '@dojo/framework/core/vdom';
-import I18nMixin, { I18nProperties } from '@dojo/framework/core/mixins/I18n';
-import ThemedMixin, { theme, ThemedProperties } from '@dojo/framework/core/mixins/Themed';
 import diffProperty from '@dojo/framework/core/decorators/diffProperty';
-import { DNode } from '@dojo/framework/core/interfaces';
 import { reference } from '@dojo/framework/core/diff';
-import { Store } from '@dojo/framework/stores/Store';
+import { DNode } from '@dojo/framework/core/interfaces';
 import Dimensions from '@dojo/framework/core/meta/Dimensions';
 import Resize from '@dojo/framework/core/meta/Resize';
-import './utils';
+import I18nMixin, { I18nProperties } from '@dojo/framework/core/mixins/I18n';
+import ThemedMixin, { ThemedProperties, theme } from '@dojo/framework/core/mixins/Themed';
+import { v, w } from '@dojo/framework/core/vdom';
+import { Store } from '@dojo/framework/stores/Store';
 
-import { Fetcher, ColumnConfig, GridState, Updater } from './interfaces';
+import * as css from '../theme/default/grid.m.css';
+import Body from './Body';
+import Footer from './Footer';
+import Header, { FilterRenderer, SortRenderer } from './Header';
+import PaginatedBody from './PaginatedBody';
+import PaginatedFooter from './PaginatedFooter';
+import { ColumnConfig, Fetcher, GridState, Updater } from './interfaces';
 import {
 	fetcherProcess,
+	filterProcess,
 	pageChangeProcess,
+	selectionProcess,
 	sortProcess,
-	filterProcess,
-	updaterProcess,
-	selectionProcess
+	updaterProcess
 } from './processes';
-
-import Header, { SortRenderer, FilterRenderer } from './Header';
-import Body from './Body';
-import Footer from './Footer';
-import PaginatedFooter from './PaginatedFooter';
-import PaginatedBody from './PaginatedBody';
-import * as css from '../theme/default/grid.m.css';
```

Comment:
This is churn from [sort-imports](https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports). Sorry about the noise.